ui/components/app/modals/edit-approval-permission/index.scss

```diff
@@ -130,27 +128,27 @@
     &__radio-button-outline--selected {
       width: 18px;
       height: 18px;
-      background: #dadcdd;
+      background: var(--color-border-default);
```

Comment:
Even though this is a background color they are overlapping 2 elements to make it look like a border 🤷